modules/manually-create-identity-access-management.adoc

```diff
@@ -198,6 +198,25 @@ spec:
 ----
 endif::google-cloud-platform[]
 
+. Create a YAML file `azure-secret.yaml` to create a secret:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-image-registry
+  name: azure-credentials
+data:
+  azure_subscription_id: <azure-subscription-id>
+  azure_client_id: <azure-client-id>
+  azure_client_secret: <azure-client-secret>
+  azure_tenant_id: <azure-tenant-id>
+  azure_resource_prefix: <azure-resource-prefix>
+  azure_resourcegroup: <azure-resourcegroup>
+  azure_region: <azure-region>
+----
+
```

Comment:
looks like step 6 and step7 are doing the same things, better to merge step6 into step7 as an example?

The example of above yaml file:
1. all value in data section need to base64 code
2. This is an example for secret from example CR object in step5, so metadata.namespce and metadata.name should come from spec.secretRef.name and spec.secretRef.namesapce in example CR object
```
metadata:
   namespace: openshift-image-registry
   name: azure-credentials
```
=> 
```
metadata:
   namespace: openshift-image-registry
   name: installer-cloud-credentials
```

Reply:
Acked. Addressed the change.